pkg/kubelet/dockershim/docker_stats.go

```diff
@@ -62,3 +63,14 @@ func (ds *dockerService) ListContainerStats(ctx context.Context, r *runtimeapi.L
 
 	return &runtimeapi.ListContainerStatsResponse{Stats: stats}, nil
 }
+
+// PodSandboxStats returns stats for a pod stats request based on pod id.
+// This function is not implemented for the dockershim.
+func (ds *dockerService) PodSandboxStats(_ context.Context, r *runtimeapi.PodSandboxStatsRequest) (*runtimeapi.PodSandboxStatsResponse, error) {
+	return nil, errors.New("Not implemented")
```

Comment:
Let's make the error a global var that consumers can check against it (it we assume that dockershim will move out of tree at some point).

Reply:
good idea